src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -588,6 +590,11 @@ public void evaluate() throws Throwable {
         }
     }
 
+    private int getTestTimeoutOverride(Description description, int def) {
```

Comment:
Minor point. I would rename `def` to `default` to make it explicit. Also it could be confused with the `def` keyword in languages like groovy.

Reply:
I don't even think you need to pass it into this method (as it's just read from `this.timeout` in the calling method anyway). You could remove the second parameter and read `this.timeout` internally in this method.